test: bluetooth: Update ICS to TCRL 2025-1

sjanc opened the pull request:
This also bumps some LE Audio spec to latest point releases.

sjanc committed:
test: bluetooth: Update ICS to TCRL 2025-1

This also bumps some LE Audio spec to latest point releases.

Signed-off-by: Szymon Janc <szymon.janc@codecoup.pl>

jhedberg commented:
@kartben @fabiobaltieri FYI this should go into 4.1 (not necessarily rc3 since it's more in the category of doc updates), but it'll need an override from some admin due to the CRLF line endings (again). I added the "Compliance: False Positive" label, but that doesn't seem to do anything as far as our automated processes are concerned.

sjanc committed:
tests: Bluetooth: Mark generated qualification file as binaries

Those are generated files and shall not be modified to match codestyle
requirements.

Signed-off-by: Szymon Janc <szymon.janc@codecoup.pl>

Thalley commented on tests/bluetooth/qualification/.gitattributes:
```diff
@@ -0,0 +1,2 @@
+*.pts binary
+*.bqw binary
```
Is it possible to add a comment in this file, stating why this is being set?
